e2e-tests/portal/cypress/e2e/journeys/checker/dashboard/dashboard.spec.js

```diff
@@ -60,13 +60,13 @@ context('View dashboard deals as a checker', () => {
       BANK1_DEALS.push(createdDeal);
     });
 
-    cy.insertOneGefApplication(GEF_DEALS.READY_FOR_CHECK, BANK1_MAKER1).then((gefDeal) => {
+    cy.insertOneGefApplication(GEF_DEALS.READY_FOR_CHECK, ADMIN).then((gefDeal) => {
       cy.setGefApplicationStatus(gefDeal._id, GEF_DEALS.READY_FOR_CHECK.status, BANK1_MAKER1).then((updatedGefDeal) => {
         BANK1_DEALS.push(updatedGefDeal.body);
       });
     });
 
-    cy.insertOneDeal(BSS_DEALS.READY_FOR_CHECK, BANK2_MAKER2);
+    cy.insertOneDeal(BSS_DEALS.READY_FOR_CHECK, BANK1_MAKER2);
```

Comment:
What's the reason for changing this from `BANK2_MAKER2` to `BANK1_MAKER2`?

Reply:
Changed back to `BANK2_MAKER2`